client/src/components/Input/LabelledInput.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styled from "styled-components";
+
+const Label = styled.label`
+  color: #425af2;
+`;
+
+export default ({ label, children, style }) => {
```

Comment:
this one doesn't really render an input but is a wrapper so maybe it can be refactores to a HOC (higher order component)  like `withLabel()` or actually embed the BaseInput component